src/main/resources/application.properties

```diff
@@ -1,2 +1,4 @@
 #spring.profiles.include=ssl
 spring.profiles.active=local,ssl
+
+server.port=8088
```

Comment:
why 8088?

Reply:
cause the 8080 is already used by the dashboard of traefik.
also i could have put ANY unused port. it really doesn't mater as it's not exposed and as it's "reverse proxyed" by traefik...